lib/rules/indent.js

```diff
@@ -1349,10 +1337,27 @@ module.exports = {
             },
 
             VariableDeclaration(node) {
-                const variableIndent = Object.prototype.hasOwnProperty.call(options.VariableDeclarator, node.kind)
+                let variableIndent = Object.prototype.hasOwnProperty.call(options.VariableDeclarator, node.kind)
                     ? options.VariableDeclarator[node.kind]
                     : DEFAULT_VARIABLE_INDENT;
 
+                const firstToken = sourceCode.getFirstToken(node),
+                    lastToken = sourceCode.getLastToken(node);
+
+                if (options.VariableDeclarator[node.kind] === "first") {
```

Comment:
Can this be simplified?

```
if (variableIndent === "first" && node.declarations.length > 1)
```

Might also allow the `let` a few lines up to revert to `const`.

Reply:
I did see that, but for some reason I thought the original declaration also covered that case. I see now that it probably doesn't. I have no issue merging this as is.